inputs/centauri/hyperspace.nix

```
@@ -1,6 +1,37 @@
 { self, ... }: {
   perSystem = { config, self', inputs', pkgs, system, crane, ... }: {
     packages = rec {
+      hyperspace-config = pkgs.writeText "config.json" ''
```

You could pass in a config object; and do 

```suggestion
      hyperspace-config = pkgs.writeText (builtins.toJSON config)
```

then fields in this config.json can be easily overridden and set in nix.

---

hyperspace uses config in TOML format.
my bad, I use the wrong extension of the file.

---

No bultins.toToml :(